app/views/groups/index.html.haml

```diff
@@ -47,4 +47,3 @@
       = t :groups_no_groups
 
 = render partial: "shared/footer"
-
```

Comment:
:+1:
you can squash this commit with the previous one or adapt the commit message.

squash with previous would be git rebase -i ba8ecf19ac9ec114f07a4d33324ddb40e84b78f2
and then replace pick with squash on the second line

change the commit message to "delete extra lines" would be
git commit --amend

Reply:
Thank you again! All suggestions implemented, hopefully all good to go now.